### PR TITLE
fix: mf-4561 click on shadowdom cause ui jumping

### PR DESCRIPTION
**Pull request by UncleBill**
https://mask.atlassian.net/browse/MF-4561

### CODE REVIEW

File: packages/flags/src/flags/index.ts [hunk 1]
```diff
@@ -13,7 +13,6 @@ export const flags = {
 
     shadowRootInit: {
         mode: '__REACT_DEVTOOLS_GLOBAL_HOOK__' in globalThis || betaOrInsiderOnly ? 'open' : 'closed',
-        delegatesFocus: true,
```
Comment (UncleBill) [inline packages/flags/src/flags/index.ts]: Critical change.

Would this break any of your intentions? @Jack-Works

Comment (Jack-Works) [reply packages/flags/src/flags/index.ts]: No

Comment (Jack-Works) [reply packages/flags/src/flags/index.ts]: hmm but you should change it case-by-case

Comment (Jack-Works) [reply packages/flags/src/flags/index.ts]: can you remove this only in the component that has the problem with this flag? cc @UncleBill 

removing this from all things might cause keyboard focus to be unusable again

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**